### PR TITLE
doc/rgw: refine "Zones" in multisite.rst

### PR DESCRIPTION
**Pull request by zdover23**
Clean up the English under the section called "Zones". Part of a larger project aimed at giving the reader more understanding earlier in the documentation of how buckets and zones and zone groups and endpoints work.

https://tracker.ceph.com/issues/58632

Signed-off-by: Zac Dover <zac.dover@gmail.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Review by anthonyeleven — APPROVED** [review]

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/49982 - Quincy backport
https://github.com/ceph/ceph/pull/49983 - Pacific backport

File: doc/radosgw/multisite.rst [hunk 1]
```diff
@@ -1283,20 +1283,21 @@ Finally, update the period.
 Zones
 -----
 
-Ceph Object Gateway supports the notion of zones. A zone defines a
-logical group consisting of one or more Ceph Object Gateway instances.
+A zone defines a logical group that consists of one or more Ceph Object Gateway
+instances. Ceph Object Gateway supports zones.
```
Comment (patrakov) [inline doc/radosgw/multisite.rst:1287]: The most important property of a zone, namely that all RGWs in the zone look into the same pools in the same cluster, is not mentioned.

Comment (zdover23) [reply doc/radosgw/multisite.rst:1287]: I will add this tomorrow, when it is not the middle of the night. It's good to see you, @patrakov.

Comment (zdover23) [reply doc/radosgw/multisite.rst:1287]: @patrakov: Can you be more specific than "look into the same pools in the same cluster"? I'm seeking a completely unambiguous sentence. Should we say that all RGWs in a given zone target a given set of pools within a given cluster?

**Comment by patrakov** [comment]
Something like this: All RGWs in a given zone serve S3 objects backed by RADOS objects in the same set of pools in the same cluster.

**Comment by zdover23** [comment]
> Something like this: All RGWs in a given zone serve S3 objects backed by RADOS objects in the same set of pools in the same cluster.

Thanks for that.

**Comment by zdover23** [comment]
> Something like this: All RGWs in a given zone serve S3 objects backed by RADOS objects in the same set of pools in the same cluster.

@patrakov, see this: https://github.com/ceph/ceph/pull/52280

**Pull request merged**